### PR TITLE
On the Utility of Learning about Humans for Human-AI Coordination

### PR DESCRIPTION
**Pull request by AntoineTheb**
Not sure if it was my turn to present this week but I found the article + blog post super interesting

### CODE REVIEW

File: article/_posts/2019-10-23-Humain-Ai-Coord.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+# Highlights
+- Self-play algorithms use (near) optimal solutions when playing together against humans
+- However, RL algorithms do not perform as well when playing _with_ humans
+- Training them to perform with humans makes them perform better with humans
+
+# Introduction
+
+> An increasingly effective way to tackle two-player games is to train an agent to play with a set of other AI agents, often past versions of itself. This powerful approach has resulted in impressive performance against human experts in games like Go, Quake, Dota, and Starcraft.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-23-Humain-Ai-Coord.md]: @jhlegarreta 
> Don't the other examples mentioned use human-human examples (e.g. two human playing Go) to learn?
No, as stated here and in the parqagraph below, all "superhuman" AIs have been trained via self-play without any human demonstration or input.

File: article/_posts/2019-10-23-Humain-Ai-Coord.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+**H3. When partnered with a human, a human-aware agent will achieve higher performance than an agent trained via imitation learning.**
+
+![](/article/images/human-ai-coord/gamedyns.png)
+![](/article/images/human-ai-coord/env.jpeg)
+
+They also created the _Overcooked_ environment, based on the video game of the same name, to measure human-agent cooperation. The environment has five different possible scenarios, as pictured above, all with the same goal: Cooking onion soup and delivering it to the serving location.
+
+> Agents should learn how to navigate the map, interact with objects, drop the objects off in the right locations, and finally serve completed dishes to the serving area. All the while, agents should be aware of what their partner is doing and coordinate with them effectively.
+
+To train the agents, the authors gathered 16 human-human trajectories for each layout. They then used behavior cloning to create the $$BC$$ model (training time) and $$H_{proxy}$$ model (test time).
```
Comment (AntoineTheb) [inline article/_posts/2019-10-23-Humain-Ai-Coord.md]: @jhlegarreta 
> Also, if they use BC, where does the real interaction/playing with humans take place? Are the human-human trajectories an improvement proposed by the authors ?

Before the actual training takes place, as stated here. Obviously, having actual humans in the training loop would be infeasible, hence using behavior cloning to mimic humans.

**Comment by AntoineTheb** [comment]
Following @jhlegarreta's comments I have added clarifications and added the link to the pdf in 
5f30e2e

File: article/_posts/2019-10-23-Humain-Ai-Coord.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+layout: review
+title: "On the Utility of Learning about Humans for Human-AI Coordination"
+tags: reinforcement deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Micah Carroll, Rohin Shah, Mark K. Ho, Thomas L. Griffiths, Sanjit A. Seshia, Pieter Abbeel, Anca Dragan"
+    title:   "On the Utility of Learning about Humans for Human-AI Coordination"
+    venue:   "NeurIPS 2019"
+pdf: ""
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply article/_posts/2019-10-23-Humain-Ai-Coord.md]: Done in 5f30e2e.

File: article/_posts/2019-10-23-Humain-Ai-Coord.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-10-23-Humain-Ai-Coord.md]: OK.

File: article/_posts/2019-10-23-Humain-Ai-Coord.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-10-23-Humain-Ai-Coord.md]: OK.

**Pull request merged**